<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>Pedro Pablo Camell√≥n Quintero</author>
 <class>ControlCultivos</class>
 <widget class="QMainWindow" name="ControlCultivos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>657</width>
    <height>377</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>108</red>
        <green>108</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>108</red>
        <green>108</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="WindowText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>108</red>
        <green>108</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Button">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Light">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Midlight">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>235</red>
        <green>235</green>
        <blue>235</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Dark">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>108</red>
        <green>108</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Mid">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>144</red>
        <green>144</green>
        <blue>144</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>108</red>
        <green>108</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="BrightText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>255</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ButtonText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>108</red>
        <green>108</green>
        <blue>108</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>191</red>
        <green>218</green>
        <blue>127</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Shadow">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="AlternateBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>216</red>
        <green>216</green>
        <blue>216</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipBase">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>255</green>
        <blue>220</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="ToolTipText">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="PlaceholderText">
      <brush brushstyle="NoBrush">
       <color alpha="128">
        <red>0</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>Mi Finca</string>
  </property>
  <property name="windowIcon">
   <iconset resource="iconos.qrc">
    <normaloff>:/ico/hoja.png</normaloff>:/ico/hoja.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <property name="iconSize">
   <size>
    <width>50</width>
    <height>50</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="QWidget" name="layoutWidget">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QPushButton" name="pB_tablaSem">
          <property name="text">
           <string>Tabla Semanal</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pB_tablaLabOb">
          <property name="text">
           <string>Labor / Obrero</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_nombreObrero">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>0</height>
           </size>
          </property>
          <property name="text">
           <string>Obrero</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cB_diaSemana">
          <property name="styleSheet">
           <string notr="true">
border-width: 2px;
border-radius: 2px;
border-color: beige;
font: bold 12px;
min-width: 4em;
padding: 2px;</string>
          </property>
          <item>
           <property name="text">
            <string>Dia (semana)</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Lunes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Martes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Miercoles</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Jueves</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Viernes</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Sabado</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cB_Labor">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
border-width: 2px;
border-radius: 2px;
border-color: beige;
font: bold 12px;
min-width: 4em;
padding: 2px;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_pago">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Pago: </string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="sB_Pago">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="sizeIncrement">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pB_AddPago">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="styleSheet">
           <string notr="true">
border-width: 2px;
border-radius: 3px;
border-color: beige;
font: bold 12px;
min-width: 4em;
padding: 2px;</string>
          </property>
          <property name="text">
           <string>A√±adir</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_errorEnDatos">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="palette">
        <palette>
         <active/>
         <inactive/>
         <disabled/>
        </palette>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>657</width>
     <height>26</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #rgb(205, 232, 136);</string>
   </property>
   <widget class="QMenu" name="menuOpciones">
    <property name="title">
     <string>Opciones</string>
    </property>
    <addaction name="actionNueva_Cosecha"/>
    <addaction name="actionGuardar_Cosecha"/>
    <addaction name="actionCompras"/>
    <addaction name="separator"/>
    <addaction name="actionSalir"/>
   </widget>
   <widget class="QMenu" name="menuAnadir">
    <property name="title">
     <string>Cultivos</string>
    </property>
    <addaction name="actionAnadir_Cultivo"/>
    <addaction name="actionEliminarCultivo"/>
   </widget>
   <widget class="QMenu" name="menuObreros">
    <property name="title">
     <string>Obreros</string>
    </property>
    <addaction name="actionAdd_Obrero"/>
    <addaction name="actionEliminar_Obrero"/>
   </widget>
   <widget class="QMenu" name="menuLabores">
    <property name="title">
     <string>Labores</string>
    </property>
    <addaction name="actionAdd_Labor"/>
    <addaction name="actionEliminar_Labor"/>
   </widget>
   <addaction name="menuOpciones"/>
   <addaction name="menuObreros"/>
   <addaction name="menuAnadir"/>
   <addaction name="menuLabores"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #cde888;</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNueva_Cosecha"/>
   <addaction name="actionGuardar_Cosecha"/>
   <addaction name="actionCompras"/>
   <addaction name="actionPagosHistoricos"/>
   <addaction name="separator"/>
   <addaction name="actionAdd_Obrero"/>
   <addaction name="actionEliminar_Obrero"/>
   <addaction name="actionCierre_Semanal"/>
   <addaction name="separator"/>
   <addaction name="actionSalir"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNueva_Cosecha">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/ico/AllDesign_biz_38.png</normaloff>:/ico/AllDesign_biz_38.png</iconset>
   </property>
   <property name="text">
    <string>Nueva Cosecha</string>
   </property>
  </action>
  <action name="actionGuardar_Cosecha">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/ico/AllDesign_biz_51.png</normaloff>:/ico/AllDesign_biz_51.png</iconset>
   </property>
   <property name="text">
    <string>Guardar Cosecha</string>
   </property>
  </action>
  <action name="actionAdd_Obrero">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/ico/new_user.png</normaloff>:/ico/new_user.png</iconset>
   </property>
   <property name="text">
    <string>Agregar Obrero</string>
   </property>
   <property name="toolTip">
    <string>Agrega un trabajador nuevo</string>
   </property>
  </action>
  <action name="actionEliminar_Obrero">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/ico/del_user.png</normaloff>:/ico/del_user.png</iconset>
   </property>
   <property name="text">
    <string>Eliminar Obrero</string>
   </property>
  </action>
  <action name="actionAdd_Labor">
   <property name="text">
    <string>A√±adir Labor</string>
   </property>
   <property name="toolTip">
    <string>Agrega una nueva labor</string>
   </property>
  </action>
  <action name="actionEliminar_Labor">
   <property name="text">
    <string>Eliminar Labor</string>
   </property>
  </action>
  <action name="actionSalir">
   <property name="text">
    <string>Salir</string>
   </property>
   <property name="toolTip">
    <string>Cierra el programa</string>
   </property>
  </action>
  <action name="actionAnadir_Cultivo">
   <property name="text">
    <string>Anadir Cultivo</string>
   </property>
  </action>
  <action name="actionCierre_Semanal">
   <property name="text">
    <string>Cierre Semanal</string>
   </property>
   <property name="toolTip">
    <string>Corte semanal</string>
   </property>
  </action>
  <action name="actionEliminarCultivo">
   <property name="text">
    <string>Eliminar Cultivo</string>
   </property>
  </action>
  <action name="actionModoNocturno">
   <property name="text">
    <string>Modo Nocturno</string>
   </property>
   <property name="toolTip">
    <string>Para que el brillo de la pantalla no moleste</string>
   </property>
  </action>
  <action name="actionImprimirCierreSemanal">
   <property name="icon">
    <iconset resource="iconos.qrc">
     <normaloff>:/ico/printer.ico</normaloff>:/ico/printer.ico</iconset>
   </property>
   <property name="text">
    <string>Imprimir Cierre Semanal</string>
   </property>
   <property name="toolTip">
    <string>Imprime un resumen de los pagos semanales</string>
   </property>
  </action>
  <action name="actionCompras">
   <property name="text">
    <string>Compras</string>
   </property>
  </action>
  <action name="actionPagosHistoricos">
   <property name="text">
    <string>Pagos Historicos</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="iconos.qrc"/>
 </resources>
 <connections/>
 <slots>
  <signal>clear()</signal>
 </slots>
</ui>
